code-the-dream-react/src/InputWithLabel.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { useEffect } from 'react';
+import { useRef } from 'react';
```

Comment:
you can just combine this two import statements into one like, `import { useEffect, useRef} from 'react'`

Reply:
Sounds good, thank you! I've also updated this upon submission of lesson 1.7!

Reply:
[Resolved in Lesson 1.7](https://github.com/fernperezg97/react-todo/pull/7)